### PR TITLE
add wingie enuygun group logo

### PR DESCRIPTION
**Pull request by yasamdogan**
Signed-off-by: Yasam Dogan <yasam.dogan@enuygun.com>

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -564,6 +564,10 @@ logo = "xbox.png"
 url = "https://zapier.com"
 logo = "zapier.png"
 
+[[params.users]]
```
Comment (zroubalik) [inline config.toml:567]: could ou please sort alphabetically?

Comment (yasamdogan) [reply config.toml]: fixed , thanks for support

**Review by zroubalik — APPROVED** [review]
Welcome 🎉

**Comment by zroubalik** [comment]
@yardenshoham please also fix DCO: 
https://github.com/kedacore/keda-docs/pull/1255/checks?check_run_id=18112531793

https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what

**Comment by yardenshoham** [comment]
Autocomplete issue of a username starting with `ya`?

**Comment by zroubalik** [comment]
> Autocomplete issue of a username starting with `ya`?

Lol, yes, sorry :)

**Review by yasamdogan — COMMENTED** [review]
Signed-off-by: Yaşam Doğan <yasam.dogan@enuygun.com>

**Review by zroubalik — APPROVED** [review]

**Comment by zroubalik** [comment]
@yasamdogan 
```
In your local branch, run: git rebase HEAD~4 --signoff
Force push your changes to overwrite the branch: git push --force-with-lease origin main
```

**Review by zroubalik — APPROVED** [review]

**Pull request merged**